src/app/containers/MostRead/utilities/testHelpers.jsx

```diff
@@ -14,7 +20,15 @@ export const setFreshPromoTimestamp = mostReadData => {
 
   // Updates first 10 promos to have a fresh date
   for (let i = 0; i < 10; i += 1) {
-    updatedMostReadData.records[i].promo.timestamp = freshDate.getTime();
+    const { records } = updatedMostReadData;
```

Comment:
Since `updatedMostReadData` is constant, we should move this line outside the loop

Reply:
Moved it outside the loop as suggested.